corpus/pages/views.py

```diff
@@ -41,3 +45,126 @@ def sig(request, sig_name):
     }
 
     return render(request, "pages/sig.html", args)
+
+
+def team(request):
+    members = ExecutiveMember.objects.filter(
+    Q(user__core__isnull=True) | Q(user__core=None)
+)
+    ieee_core_posts = [
+        "Convenor", 
+        "Chairperson", 
+        "Vice Chairperson",
+        "Secretary",
+        "Joint Secretary",
+        "Treasurer(Branch)",
+        "Treasurer(Institute)",
+        "Webmaster",
+        "Media Lead",
+        "Outreach Lead",
+        "Envision Lead",
+        "Labs Lead",  
+        ]
+    
+
+    compsoc_core_posts = [
+        "CompSoc Chair",
+        "CompSoc Vice Chair",
+        "CompSoc Secretary",
+        "CompSoc Project Head",
+        "CompSoc Project Coordinator",
+        "CIS Chair",
+        "CIS Secretary",
+        "CIS Project Head",
+
+    ]
+
+    diode_core_posts = [
+        "Diode Chair",
+        "SPS Chair",
+        "SPS Vice Chair",
+        "SPS Secretary",
+        "CAS Chair",
+        "CAS Vice Chair",
+        "CAS Secretary",
+        "RAS Chair",
+        "RAS Secretary",
+    ]
+
+    piston_core_posts = [
+        "Piston Chair",
+        "Piston Vice Chair",
+        "Piston Secretary",
+        "Piston Project Head",
+        "IAS Chair",
+        "IAS Secretary",
+    ]
```

Comment:
All this isn't needed right? Why can't we just filter by the `sig` field in the `Core` model?
Here's my understanding:
- All Core post holders have a "SIG" (for lack of a better term). SIG Cores will obviously have their SIG value as their SIG name. ExeCom will have their SIG value as `execom` (or whatever else suits). SAC Core will have their SIG value as `sac` (or whatever else suits).
- In the `filter` statements below, just use the SIG value instead of checking if a post is in this list or not.

Also, this logic can (and should) be moved into the `models.py` file. I've added a comment in that file as well to make the `post` field an enum. You can probably use these values directly in making the `choices` list.

Reply:
Just for clarification, ExeCom means IEEE core, right? So core members who are IEEE core have their sig reassigned to ExeCom? Similarly, what is SAC? (Sorry but I have very little idea how IEEE core structure works rn XD )

Reply:
@yukitya-1811 no issues, that's probably an oversight on my part. There should be a core structure and responsibilities doc, maybe @anirudhprabhakaran3 can guide you to it.
Yes, ExeCom (Executive Committee) is the same as what you have defined as IEEE Core here.
Core members who are IEEE Core will have the SIG field in the Core table as ExeCom.
SAC is the Student Activity Committee. These core posts include the media head, outreach lead, Envision head, etc. You should find all these details in the Core Structure and Responsibilities doc.